packages/backend/src/tests/outgoingPayment.ts

```diff
@@ -36,5 +36,9 @@ export async function createOutgoingPayment(
     quoteId: quote.id
   })
   assert.ok(!isOutgoingPaymentError(outgoingPaymentOrError))
+
+  const accountingService = await deps.use('accountingService')
+  await accountingService.createLiquidityAccount(outgoingPaymentOrError)
+
```

Comment:
Unless there's tests tinkering with the balance, I think this isn't needed
```suggestion

  const accountingService = await deps.use('accountingService')
  await accountingService.createLiquidityAccount(outgoingPaymentOrError)

```

Reply:
Some test go Straight from `Funding` state to `Completed` or `Failed` without calling `fund`. So the Tigerbeetle account is never created via the normal flow. This is to accommodate those tests.